demos/html5/froala3/test/e2e-insert-formula.test.js

```diff
@@ -34,6 +34,7 @@ describe('Insert Formula. TAG = Insert',
 
     // Execute before each test of the file to open the demo page
     beforeEach(async () => {
+      jest.setTimeout(30000);
```

Comment:
This seems like a very long timeout wait for test: 30 seconds.
Is there a way to avoid waiting that long?